indigo_app/templates/place/detail.html

```diff
@@ -86,6 +86,11 @@
               {{/if}}
               <br>
               <span class="text-muted">{{ frbr_uri }}</span>
+              {{#if parent_work}}
+              <p>
```

Comment:
`p` has bottom padding, you probably just want a `div`.

Can you include a screenshot?